chevah_build

```diff
@@ -202,6 +202,9 @@ case $OS in
         export CXX="clang++"
         export CFLAGS="$CFLAGS -mmacosx-version-min=10.8"
         export MACOSX_DEPLOYMENT_TARGET=10.8
+        # The location of the Homebrew OpenSSL libs and include directories.
+        export LDFLAGS="$LDFLAGS -L/usr/local/opt/openssl/lib"
```

Comment:
is this still for 10.8 ?

Why homebrew and not the OpenSSL version provided by OSX?